FlashFeetAPI/src/main/java/org/example/api/JobRoleService.java

```diff
@@ -0,0 +1,45 @@
+package org.example.api;
+
+import org.example.cli.JobRole;
+import org.example.client.FailedToDeleteJobRoleException;
+import org.example.client.FailedToGetJobsException;
+import org.example.client.JobRoleDoesNotExistException;
+import org.example.db.DatabaseConnector;
+import org.example.db.JobRoleDao;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+
+public class JobRoleService {
+
+    private JobRoleDao jobRoleDao = new JobRoleDao();
+
+    public DatabaseConnector databaseConnector = new DatabaseConnector();
+
+    public JobRoleService(){
+
+    }
+
+    public JobRoleService(JobRoleDao jobRoleDao, DatabaseConnector databaseConnector) {
+        this.jobRoleDao = jobRoleDao;
+        this.databaseConnector = databaseConnector;
+    }
+
+    public List<JobRole> getAllJobRoles() throws FailedToGetJobsException, SQLException {
+        return jobRoleDao.getAllJobRoles(databaseConnector.getConnection());
+    }
+
+    public int deleteJobRole(String jobRole, DatabaseConnector databaseConnector) throws JobRoleDoesNotExistException, FailedToDeleteJobRoleException {
+        try {
+            JobRole deleteJobRole = jobRoleDao.getJobRole(jobRole);
```

Comment:
This is not being used. Do you want to add a check in here to see if the jobRole exist first?